modules/removing-custom-config-from-machinepool.adoc

```diff
@@ -21,13 +21,13 @@ To remove all `KubeletConfig` objects from the machine pool, set an empty value
 +
 [source,terminal]
 ----
-$ rosa edit machinepool -c <cluster_name> --kubeletconfigs="" --name <machinepool_name>
+$ rosa edit machinepool -c <cluster_name> <machinepool_name> --kubeletconfigs=""
```

Comment:
@laubai I'd suggest keeping the `<machinepool_name>` at the end of the command for consistency:

```
rosa edit machinepool -c <cluster_name> --kubeletconfigs="" <machinepool_name>
```